bgpd/bgp_vty.c

```diff
@@ -624,18 +624,12 @@ static int bgp_clear(struct vty *vty, struct bgp *bgp, afi_t afi, safi_t safi,
 
 			if (ret < 0)
 				bgp_clear_vty_error(vty, peer, afi, safi, ret);
-			else
-				found = true;
 		}
 
 		/* This is to apply read-only mode on this clear. */
 		if (stype == BGP_CLEAR_SOFT_NONE)
 			bgp->update_delay_over = 0;
 
-		if (!found)
-			vty_out(vty, "%%BGP: No %s peer configured\n",
-				afi_safi_print(afi, safi));
-
```

Comment:
is there an reason to *not* remove the  found variable from the routine completely, i.e., drop line 605?

Reply:
donald says compiler complained when removed, I belive him ;-)